lib/elixir/lib/supervisor.ex

```diff
@@ -501,9 +502,15 @@ defmodule Supervisor do
           | {:ok, child, info :: term}
           | {:error, {:already_started, child} | :already_present | term}
 
+  @typedoc """
+  A child process.
+
+  It can be PID when a process was started, or `:undefined` when a
+  child was created by a [dynamic supervisor](`DynamicSupervisor`).
```

Comment:
Typos

```suggestion
  It can be a PID when the child process was started, or `:undefined` when
  the child was created by a [dynamic supervisor](`DynamicSupervisor`).
```